crowbar_framework/config/initializers/delayed_job_config.rb

```diff
@@ -14,12 +14,27 @@
 # limitations under the License.
 #
 
+module Delayed
+  class Worker
+    @@log_file = if Rails.env.production?
```

Comment:
I dont know why but it didnt work on my tests if I was using a class instance var. Probably was doing something wrong here, so I need to recheck :/